DESCRIPTION

```diff
@@ -18,7 +18,7 @@ URL: https://github.com/esqLABS/esqlabsR
 BugReports: https://github.com/esqLABS/esqlabsR/issues
 Depends:
     R (>= 4.1),
-    ospsuite (>= 11.1),
+    ospsuite (>= 11.2.224),
```

Comment:
Ok let's not do this... It will not work any more if using `ospsuite` built locally, because the version there is `11.1` or `11.2` but is not synchronized with the AppVeyor version.

Reply:
Ok, i'll try to think of something to make dependency on specific more secure.